src/utils/utils.ts

```diff
@@ -386,10 +388,17 @@ export async function createRedis(serverConfig: PluginsServerConfig): Promise<Re
         ...credentials,
         maxRetriesPerRequest: -1,
     })
+    let errorCounter = 0
     redis
         .on('error', (error) => {
+            errorCounter++
             Sentry.captureException(error)
-            status.error('🔴', 'Redis error encountered! Trying to reconnect...\n', error)
+            if (errorCounter > REDIS_ERROR_COUNTER_LIMIT) {
+                status.error('😡', 'Redis error encountered! Enough of this, I quit!\n', error)
+                killGracefully()
+            } else {
+                status.error('🔴', 'Redis error encountered! Trying to reconnect...\n', error)
```

Comment:
The only "nit" (it's not even that) I'd have is that this could tell me how many more retries to connect it'll perform. Might not even be super useful though.

Reply:
I'll table that for now